lib/jekyll/multiple/languages/plugin.rb

```diff
@@ -73,136 +245,239 @@ def permalink
+          return                                    "Include file '#{file}' contains invalid characters or sequences"
         end
 
         Dir.chdir(includes_dir) do
           choices = Dir['**/*'].reject { |x| File.symlink?(x) }
+          
           if choices.include?(file)
-            source = File.read(file)
+            source  = File.read(file)
             partial = Liquid::Template.parse(source)
 
             context.stack do
-              context['include'] = parse_params(context) if @params
-              contents = partial.render(context)
-              site = context.registers[:site]
-              ext = File.extname(file)
+              context['include'] = parse_params(  context) if @params
+              contents           = partial.render(context)
+              site               = context.registers[:site]
+              ext                = File.extname(file)
 
               converter = site.converters.find { |c| c.matches(ext) }
-              contents = converter.convert(contents) unless converter.nil?
+              contents  = converter.convert(contents) unless converter.nil?
 
               contents
             end
           else
-            "Included file '#{file}' not found in #{includes_dir} directory"
+            puts "Jekyll Multiple Languages: ".bold + "Included file '#{file}' not found in #{includes_dir} directory".red
+            return                                    "Included file '#{file}' not found in #{includes_dir} directory"
           end
         end
       end
     end
   end
 
+
+
+  ##############################################################################
+  # class LocalizeLink
+  #
+  # Creates links or permalinks for translated pages.
+  # User must use the "tl" or "translate_link" liquid tags.
+  ##############################################################################
   class LocalizeLink < Liquid::Tag
 
+    #======================================
+    # initialize
+    #======================================
     def initialize(tag_name, key, tokens)
       super
       @key = key
     end
-
+    
+    
+    
+    #======================================
+    # render
+    #======================================
     def render(context)
-      if "#{context[@key]}" != "" #Check for page variable
+      if      "#{context[@key]}" != "" # Check for page variable
         key = "#{context[@key]}"
       else
         key = @key
       end
-      key = key.split
-      namespace = key[0]
-      lang = key[1] || context.registers[:site].config['lang']
-      default_lang = context.registers[:site].config['default_lang']
-      baseurl = context.registers[:site].baseurl
-      pages = context.registers[:site].pages
-      url = "";
-
+      
+      key          = key.split
+      namespace    = key[0]
+      lang         = key[1] || context.registers[:site].config[        'lang']
+      default_lang =           context.registers[:site].config['default_lang']
```

Comment:
Not a fan of this kind of indentation ... If we need to add a new variable like "default_something", it will break the layout because it's longer, we will then need to add extra spaces in lines 387 to 393, thus making multiple lines change in the commit instead of only 1 new line. We lose tracking of real changes.